docs/conf/portland/2019/news/announcing-speakers.rst

```diff
@@ -0,0 +1,137 @@
+:template: {{year}}/generic.html
+
+.. post:: Feb 27, 2019
+   :tags: portland-2019, cfp, talks
+
+Speaker Announcement
+====================
+
+Happy Wednesday everybody!
+
+We’re less than three months away from Write the Docs Portland 2019
+conference! We have so much news to share, so without further adieu,
+here we go!
+
+First and foremost, **we’re very excited to announce our finalized speaker
+list for the conference!** Our speaker lineup this year is packed with a
+diverse and compelling melange of documentarians who have amazing
+stories to share with conference attendees. We feel that with our
+speakers – whose expertise spans across many fields and industries –
+will share interesting and actionable takeaways for attendees.
+
```

Comment:
I'd put the speaker list here, it'll help to separate out the big chunks of text